bin/wasm-node/javascript/src/index.js

```diff
@@ -79,11 +106,40 @@ export async function start(config) {
       // `expected` was pushed by the `addChain` method.
       // Resolve the promise that `addChain` returned to the user.
       expected.resolve({
+        health: () => {
+          // are we already waiting for a health message reply?
+          if (chainHealthRequestCallbacks[chainId]) {
+            const reject = chainHealthRequestCallbacks[chainId].reject;
+            delete chainHealthRequestCallbacks[chainId];
+            return reject(new Error('Called health before pending health request replied'));
+          }
+
+          // craft a new system_health messsage
+          const id = uuidv4();
+          const request = `{"id":"${id}","jsonrpc":"2.0","method":"system_health","params":[]}`;
```

Comment:
I'd prefer not doing weird string constructions.

```suggestion
          const request = JSON.stringify({ id, jsonrpc: "2.0", method: "system_health", params: [] });
```